rllib/utils/error.py

```diff
@@ -11,3 +11,50 @@ class UnsupportedSpaceException(Exception):
 class EnvError(Exception):
     """Error if we encounter an error during RL environment validation."""
     pass
+
+
+# -------
+# Error messages
+# -------
+
+# Message explaining there are no GPUs available for the
+# num_gpus=n or num_gpus_per_worker=m settings.
+ERR_MSG_NO_GPUS = \
+    """Found {} GPUs on your machine (GPU devices found: {})! If your machine
```

Comment:
nit, why repeat # of devices found in the ()?

Reply:
In the (), we report the actual device names found, e.g. ["/gpu:0", "/gpu:1"].

I wanted to avoid that the user sees:
`Found GPUs=[]` and then wondering what that `[]` means.